internal/runbits/buildscript/file.go

```diff
@@ -47,24 +48,24 @@ type primeable interface {
 	primer.SvcModeler
 }
 
-func Initialize(path string, auth *authentication.Auth, svcm *model.SvcModel) error {
-	scriptPath := filepath.Join(path, constants.BuildScriptFileName)
+func Initialize(proj projecter, auth *authentication.Auth, svcm *model.SvcModel) error {
```

Comment:
I know that we would prefer to not make buildscripts aware of projects, but since we're not having a primed checkoutinfo anymore, this is the next best thing. I could pass dir, owner, name, branch if you want though.